.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -120,10 +120,19 @@ body:
         - Fabric (New Architecture)
 
   - type: dropdown
-    id: build-type
+    id: bundle-build-type
     attributes:
-      label: Build type
-      description: What mode your application is running?
+      label: JavaScript bundle build type
+      description: What is the type of your JavaScript bundle?
```

Comment:
People will have no idea what we are asking about, we need to word it differently

Reply:
I didn't come up with how to phrase it better but keep in mind that it's an optional field.

Reply:
The official term is "bundle mode".